cmd/compgen/.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Run Compgen",
+      "type": "go",
+      "request": "launch",
+      "mode": "auto",
+      "program": "${workspaceFolder}",
+      "args": [
```

Comment:
Are these args updated with the latest changes? I'm thinking the path for this should be a debug path. IIRC VSCode creates a .debug. I don't think we'd want a real receiver being created in the directory when trying to debug. It could be accidentally committed.

Reply:
It is update for the latest changes like removing the pkg directory. But this is a good suggestion about using an alternative path. Are you thinking of a top level directory named "debug" or something similar?

Reply:
i think vscode auto creates a .debug folder when running, so you could just use that or the .vscode. wherever you choose to do it, it should be ignored and not impact the rest of the code.

Reply:
I couldn't find this functionality. But it was pretty easy to build into `.vscode/launch.json` and `.vscode/tasks.json`. If you can, please give this commit a review as well.